server/standalone/migration/000045_remove_delete_intermediary_model.down.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE compute_plans
+ADD COLUMN IF NOT EXISTS delete_intermediary_models bool DEFAULT false;
```

Comment:
Can't we add the `not null` constraint in a single statement?
```suggestion
ADD COLUMN IF NOT EXISTS delete_intermediary_models bool NOT NULL DEFAULT false;
```

Reply:
IIRC no, but I changed a lot of stuff since I tested so maybe we can retry that.

Reply:
Looks like it work and it was some other part of my request that was broken.